test/CMakeLists.txt

```diff
@@ -1,4 +1,6 @@
 add_subdirectory(trace_files)
 
-add_subdirectory(plugins)
-add_subdirectory(confs/plugins)
+if(NOT MUSL_OPTIMIZED_BUILD)
```

Comment:
in musl build we don't have plugins so I've disabled the testing phase for plugins